File: network_cs/Services/IFestivalService.cs
using Dtos;
using System.Collections.ObjectModel;


namespace Services
{
    public interface IFestivalService
    {
        void Login(AngajatDto angajat,IFestivalObserver client);
        void Logout(AngajatDto angajat);
        Collection<ConcertDto> GetConcerts();
        void SellTickets(ConcertDto concert,int nrTickets);
    }
}

File: network_java/FestivalServices/src/main/java/services/IFestivalService.java
package services;

import dtos.AngajatDto;
import dtos.ConcertDto;

import java.util.Collection;

public interface IFestivalService {

    void login(String username, String password,IFestivalObserver client) throws FestivalException;
    void logout(AngajatDto angajat) throws FestivalException;
    Collection<ConcertDto> getConcerts() throws FestivalException;
    void sellTickets(ConcertDto concert, int nrBilete) throws FestivalException;

}
